tools/scoring/score/dimensions/entity_identification.py

```diff
@@ -30,38 +30,39 @@ class EntityIdentification(Dimension):
   def __init__(self, *, deserialized_files: DeserializedFilesDict):
     super().__init__(deserialized_files=deserialized_files)
 
-    proposed, solution = map(deserialized_files.get, (PROPOSED, SOLUTION))
+    proposed_file, solution_file = map(deserialized_files.get,
+                                       (PROPOSED, SOLUTION))
 
     # Lists of `cloud_device_id`s representing
     # reporting entities with canonical types
     solution_reporting = [
         entity.cloud_device_id for entity in filter(
             self.is_entity_canonical,
```

Comment:
this is the same code, you can use here a helper method, listCloudDeviceId might not be the most appropriate.

```
_listCloudDeviceId(something):
[
        entity.cloud_device_id for entity in filter(
            self.is_entity_canonical,
            filter(self.is_entity_reporting, something.values()))
    ]

```

then you can use it like this:

```
solution_reporting = _listCloudDeviceId(solution_file)
proposed_reporting = _listCloudDeviceId(proposed_file)
```